src/test/java/uk/gov/companieshouse/api/accounts/validation/DebtorsValidatorTest.java

```diff
@@ -136,6 +127,18 @@ void tesInvalidMultipleYearDebtorsNote() {
             new Error("inconsistent_data", PREVIOUS_TRADE_DEBTORS,
                 LocationType.JSON_PATH.getValue(),
                 ErrorType.VALIDATION.getType())));
+        assertTrue(errors.containsError(
+            new Error("inconsistent_data", PREVIOUS_PREPAYMENTS,
+                LocationType.JSON_PATH.getValue(),
+                ErrorType.VALIDATION.getType())));
+        assertTrue(errors.containsError(
+            new Error("inconsistent_data", PREVIOUS_GREATER_THAN_ONE_YEAR,
+                LocationType.JSON_PATH.getValue(),
+                ErrorType.VALIDATION.getType())));
+        assertTrue(errors.containsError(
+            new Error("inconsistent_data", PREVIOUS_OTHER_DEBTORS,
+                LocationType.JSON_PATH.getValue(),
+                ErrorType.VALIDATION.getType())));
```

Comment:
we could move this to a method as it will reduce the lines of code and we will isolate the error creation to one place only. 

```
  assertTrue(errors.containsError(createError("inconsistent_data", PREVIOUS_TOTAL_PATH);
```
where: 
```
    private Error createError(String error, String path) {
        return new Error("inconsistent_data", PREVIOUS_OTHER_DEBTORS,
            LocationType.JSON_PATH.getValue(),
            ErrorType.VALIDATION.getType());
    }
```